Cargo.toml

```diff
@@ -86,6 +86,7 @@ termcolor.workspace = true
 thiserror.workspace = true
 threadpool = "1"
 tokio = { workspace = true, optional = true }
+tiny_http = "0.11"
```

Comment:
Given that we're already depending on hyper for the reqwest download backend, I think we should be using hyper (0.14) for the server, too.

Reply:
Too easy, will do